src/core/sys/posix/stdio.d

```diff
@@ -252,3 +252,9 @@ else version( solaris )
 
     char*  tempnam(in char*, in char*);
 }
+
+version (Posix)
```

Comment:
ugh! Strange that the former symbols are not in a `src/core/sys/linux/stdio.d`...